tools/miaou.ml

```diff
@@ -147,9 +147,14 @@ module Make
 
     let next_indent indent = indent ^ "  "
 
-    let next_event = function
-      | 1 -> 3
-      | n -> n+1
+    module Next : sig
+      val reset : unit -> unit
+      val next : unit -> int
+    end = struct
+      let c = ref 3
+      let reset () = c := 3
```

Comment:
should we start from 1 instead and make a call to `next ()` in lines 672, 673, 677? That way miaou7 will also handle correctly the indices for definitions of events like
```
let evts1 = range([M]; po; [M])
```

Reply:
Thanks for your suggestion. Will fix now,.